PSModule/Helper.cs

```diff
@@ -285,11 +286,10 @@ public static int GetNumberOfTests(IList<ReportMetaData> listReport, out IDictio
                 table.RenderControl(new HtmlTextWriter(sw));
                 html = sw.ToString();
             }
-            File.WriteAllText($@"{uftWorkingFolder}\res\Report_{buildNumber}\UFT Report", html);
+            File.WriteAllText($@"{rptPath}\{UFT_REPORT}", html);

```

Comment:
It is recommended to use `Path.Combine` method instead of concatenating the path by string.